docs/RESOURCES.md

```diff
@@ -1,13 +1,8 @@
 # Resources
 
-<!-- TODO: write document
-
-  This document should link to useful resources.
-
-  Feel free to link to any good resources you know, whether they
-  be websites, videos, mailing lists, etc.
-
-  The contents of this document are displayed on the track's documentation
-  page at `https://exercism.org/docs/tracks/<track>/resources`.
-
-  See https://exercism.org/docs/building/tracks/docs for more information. -->
+* [Official Godot Homepage](https://godotengine.org/)
```

Comment:
@BNAndras could you please also add a direct link to the documentation?

Reply:
Thank you for adding the link! However, what I meant is to have both a link to the website and a link to documentation ;)